docs/reference-architectures/serverless/web-app.md

```diff
@@ -166,29 +155,34 @@ For more information, see [Working with client identities](/azure/azure-function
 
 ### CORS
 
-In this reference architecture, the web application and the API do not share the same origin. That means when the application calls the API, it is a cross-origin request. Browser security prevents a web page from making AJAX requests to another domain. This restriction is called the *same-origin policy* and prevents a malicious site from reading sensitive data from another site. To enable a cross-origin request, add a Cross-Origin Resource Sharing (CORS) [policy][cors-policy] to the API Management gateway:
-
-```xml
-<cors allow-credentials="true">
-    <allowed-origins>
-        <origin>[Website URL]</origin>
-    </allowed-origins>
-    <allowed-methods>
-        <method>GET</method>
-    </allowed-methods>
-    <allowed-headers>
-        <header>*</header>
-    </allowed-headers>
-</cors>
+Front Door sits in front of both the web application and the API. The primary domain points at Front Door, not the application pieces themselves. As such, it is possible to direct traffic going to domain.com to the websites and traffic to domain.com/api to the API. This means that both pieces of the application will share the same origin, and cross-origin considerations will not be necessary
+
+In development, the website and API are two different services running on two different origins. That means when the application calls the API, it is a cross-origin request. Browser security prevents a web page from making AJAX requests to another domain. This restriction is called the *same-origin policy* and prevents a malicious site from reading sensitive data from another site. To enable a cross-origin request, specify the development host and port for the application in the CORS section of the `local.settings.json` file which is located in your function application. 
+
+```json
+{
+  "Values": {
+  },
+  "Host": {
+    "CORS": "http://localhost:3000"
+  }
+}
 ```
 
+**It is important that you remove this setting before publishing to production so that your API is not open to any user making a call from the location specified in the "CORS" setting.** 
```

Comment:
Maybe we don't need this section, but CORS is definitely needed in development and it's a tad confusing if you don't know that.